src/components/tl/mlx5/tl_mlx5_team.c

```diff
@@ -204,7 +194,7 @@ ucc_status_t ucc_tl_mlx5_team_get_scores(ucc_base_team_t *  tl_team,
         MAX_MSG_SIZE * UCC_TL_TEAM_SIZE(team), UCC_TL_MLX5_DEFAULT_SCORE,
         ucc_tl_mlx5_coll_init, tl_team);
     if (UCC_OK != status) {
-        tl_error(lib, "failed to add range to score_t");
+        tl_debug(lib, "failed to add range to score_t");
```

Comment:
how do we decide that UCC_MEMORY_TYPE_CUDA is supported?

Reply:
We need to test it in mtt and add a check for support of libibverbs on GPU direct
I leave it as is for now since I plan to work on adding those tests in a near future